mlir/include/air/Dialect/AIR/AIR.td

```diff
@@ -347,8 +347,8 @@ def air_DmaMemcpyNdOp: air_Op<"dma_memcpy_nd",
   let results = (outs Optional<air_AsyncToken>:$async_token);
   let assemblyFormat = [{
     custom<AsyncDependencies>(type($async_token), $async_dependencies)
-    `(` $dst `[` ($dst_offsets^)? `]``[` ($dst_sizes^)? `]``[` ($dst_strides^)? `]` `,`
-        $src `[` ($src_offsets^)? `]``[` ($src_sizes^)? `]``[` ($src_strides^)? `]` `)`  attr-dict `:`
+    `(` $dst (`<` $dst_offsets^ `>`)? (`<` $dst_sizes^ `>`)? (`<` $dst_strides^ `>`)? `,`
+        $src (`<` $src_offsets^ `>`)? (`<` $src_sizes^ `>`)? (`<` $src_strides^ `>`)? `)`  attr-dict `:`
```

Comment:
Does this allow 1 or 2 of the three fields? What happens then? Maybe some builder or verifier checks are needed.

Reply:
This is a good point. If only one or two of the three fields remain, then it would be difficult to tell what each remaining field represents.